pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -1184,7 +1184,7 @@ contract TimelockAuthorizer is IAuthorizer, ReentrancyGuard {
         );
 
         for (uint256 i = 0; i < executors.length; i++) {
-            // Note that we allow for repeated executors - this is not an issue
+            require(!_isExecutor[scheduledExecutionId][executors[i]], "ALREADY_AN_EXECUTOR");
```

Comment:
```suggestion
            require(!_isExecutor[scheduledExecutionId][executors[i]], "DUPLICATE_EXECUTORS");
```

I think this is slightly more explicit - they weren't 'already' an executor (i.e. before scheduling), but rather they sent the same one multiple times